### PR TITLE
CellSpurs: Implement Spooktober functions (part 2)

### PR DESCRIPTION
**Pull request by elad335**

### CODE REVIEW

**Comment by elad335** [comment]
More functions.

File: rpcs3/Emu/Cell/Modules/cellSpurs.cpp [hunk 1]
```diff
@@ -4394,24 +4533,169 @@ s32 _spurs::check_job_chain_attribute(u32 sdkVer, vm::cptr<u64> jcEntry, u16 siz
...
+		default: return error;
+		}
+	};
+
+	if (auto err = ppu_execute<&cellSpursWaitForWorkloadShutdown>(ppu, +jobChain->spurs, wid))
+	{
+		return as_job_error(err);
+	}
+
+    if (auto err = ppu_execute<&cellSpursRemoveWorkload>(ppu, +jobChain->spurs, wid))
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/Modules/cellSpurs.cpp:4689]: Indentation

Comment (elad335) [reply rpcs3/Emu/Cell/Modules/cellSpurs.cpp]: Fixed.

**Comment by Nekotekina** [comment]
LGTM

**Pull request merged**